dashboard/test/factories/pd_factories.rb

```diff
@@ -204,10 +219,22 @@
 
         workshop.facilitators.each do |facilitator|
           Pd::TeacherconSurvey.facilitator_required_fields.each do |field|
+            if Pd::TeacherconSurvey.options.key? field
+              answers = Pd::TeacherconSurvey.options.key? field
+              survey_hash[field][facilitator.name] = evaluator.randomized_survey_answers ? answers.sample : answers.last
+            else
+              survey_hash[field][facilitator.name] = evaluator.randomized_survey_answers ? SecureRandom.hex[0..8] : 'Free Response'
+            end
             survey_hash[field][facilitator.name] = 'Free response'
           end
         end
 
+        if Pd::TeacherconSurvey::DISAGREES.include?(survey_hash['personalLearningNeedsMet'])
+          survey_hash[:how_could_improve] = 'Rant about how to improve things'
```

Comment:
any reason why the other free responses get randomized hex values and this one doesn't?